localstack/services/ses/ses_starter.py

```diff
@@ -66,9 +69,20 @@ def email_responses_send_email(self):
 
     email_responses.send_email = email_responses_send_email
 
+    email_responses_list_templates_orig = email_responses.list_templates
+
+    def list_templates(self):
+        email_templates = ses_backend.list_templates()
+        for template in email_templates:
+            if isinstance(template['Timestamp'], (date, datetime)):
+                # Hack to change the last digits to Java SDKv2 compatible format
+                template['Timestamp'] = template['Timestamp'].isoformat()[:-3] + 'Z'
```

Comment:
nitpick: I think we could probably use the `timestamp_millis` util function from `utils/common.py` here:
```
template['Timestamp'] = timestamp_millis(template['Timestamp'])
```